components/dashboard/src/components/react-datepicker.css

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+.react-datepicker-wrapper {
```

Comment:
**suggestion**: Last comment, promise! If it's not too much to add, `rounded-xl` on the calendar element!

| BEFORE | AFTER |
|-|-|
| <img width="632" alt="calendar-before" src="https://user-images.githubusercontent.com/120486/201180520-80243ead-31be-4b09-b049-295d468cc079.png"> | <img width="632" alt="calendar-after" src="https://user-images.githubusercontent.com/120486/201180528-82a2520f-e793-4d06-ab35-8627b9d56295.png"> |